cmd/state/internal/cmdtree/hello_example.go

```diff
@@ -0,0 +1,75 @@
+package cmdtree
```

Comment:
Please exclude the example stuff from being built on CI. We don't want to ship these, just have them serve as code examples when developing locally.